README.md

```diff
@@ -50,6 +50,25 @@ mcpim ./example.py
 ```
 This will produce an `example.mcpi` mod file.
 
+## Troubleshooting
+If you get this error: module not found: _tkinter or something of the sort, you can use homebrew and use a custom tap made by @gamer4life1.
+```shell
+# First, install brew (if you don't have it already (make sure to install in /home/linuxbrew/.linuxbrew
```

Comment:
Here you have a few typos: unmatching `()`

Reply:
Ah oops